Videos/Smart Configurator Videos_RL78.htm

```diff
@@ -117,6 +118,16 @@ <h3>Smart Configurator Feature - Easy Transfer of Component Settings from One Pr
                         </div>
                         <a href="https://www.renesas.com/video/smart-configurator-feature-easy-transfer-component-settings-one-project-another">View on Renesas website</a>
                     </div>
+                    <div id="_Toc87439360" class="video-contents">
+                        <iframe src="https://players.brightcove.net/5260471205001/RyxO3haSE_default/index.html?videoId=6314306435112" 
+                        allowfullscreen=allowfullscreen></iframe>
+                        <h3>Easy Coding in e² studio With the Smart Configurator Developer Assistance Feature</h3>
+                        <div class="video-description">
+                            <p>This video introduces the Smart Configurator Developer Assistance feature that helps you code with ease, improving software development efficiency.</p>
+                            <p>Date of upload: 2022-10-25</p>
+                        </div>
+                        <a href="https://www.renesas.com/sg/en/video/easy-coding-e-studio-smart-configurator-developer-assistance-feature">View on Renesas website</a>
```

Comment:
Remove "sg/en" from the HTML links. 
This will ensure users be their own region will see the Videos page in their own language and regional website.

Reply:
Updated